nanofi/src/api/nanofi.cpp

```diff
@@ -89,6 +93,39 @@ nifi_instance *create_instance(const char *url, nifi_port *port) {
   return instance;
 }
 
+standalone_processor *create_processor(const char *name) {
+  static int proc_counter = 0;
+  auto ptr = ExecutionPlan::createProcessor(name, name);
+  if (!ptr) {
+    return nullptr;
+  }
+  if (standalone_instance == nullptr) {
+    nifi_port port;
```

Comment:
this is an input port identifier.

Reply:
Created with dummy value. Do you think having a valid input port is valid usecase for an instance we don't even expose? (As we use it to "store" standalone processors)